schemas/WebpackOptions.json

```diff
@@ -3380,23 +3380,23 @@
               "description": "Logical AND.",
               "oneOf": [
                 {
-                  "$ref": "#/definitions/RuleSetConditions"
+                  "$ref": "#/definitions/RuleSetCondition"
                 }
               ]
             },
             "not": {
               "description": "Logical NOT.",
               "oneOf": [
                 {
-                  "$ref": "#/definitions/RuleSetConditions"
+                  "$ref": "#/definitions/RuleSetCondition"
                 }
               ]
             },
             "or": {
               "description": "Logical OR.",
               "oneOf": [
                 {
-                  "$ref": "#/definitions/RuleSetConditions"
+                  "$ref": "#/definitions/RuleSetCondition"
                 }
               ]
             }
```

Comment:
Could you put the whole object with and, or, not into a separate definition `RuleSetLogicalConditions`.

This helps in generating more readable typings.